naga/src/front/wgsl/lower/mod.rs

```diff
@@ -521,6 +521,16 @@ impl<'source, 'temp, 'out> ExpressionContext<'source, 'temp, 'out> {
         }
     }
 
+    fn get(&self, handle: Handle<crate::Expression>) -> &crate::Expression {
```

Comment:
I'm not sure about this. Is this kind of direct access to the expressions something that was intentionally avoided, or was it just not needed before?

Reply:
I think it's a bit of both: It's easy to pass a `Handle` from an arena and try to index a different arena with it. It's also usually not necessary to have this since we usually know which arena to index.